cli/internal/providers/retl/sqlmodel/model.go

```diff
@@ -20,8 +20,8 @@ const (
 	SQLKey              = "sql"
 	IDKey               = "id"
 	SourceTypeKey       = "source_type"
-	CreatedAtKey        = "created_at"
-	UpdatedAtKey        = "updated_at"
+	CreatedAtKey        = "createdAt"
```

Comment:
I don't like the the fact that these are camelCase while the rest are snake case. I get this is done because of how lister puts createdAt/updateAt at the top. This is a nit-pick though, we can figure out how listing is displayed consistently in another PR

Reply:
YAMLs should use snake_case for sure.
I'm not sure about CLI in general. Without state, this serialization becomes irrelevant for internal purposes. It only plays a role when we display properties, either in diffs or in these lists. We could stick to the fields API responses have when we list them. We'll figure it out in the polish task we'll pick up before we move to beta.